src/executePrettier.js

```diff
@@ -15,10 +15,11 @@ const {
 const EMBEDDED_JS_REGEX = /<script\b[^>]*>([\s\S]*?)(?=<\/script>)/gi;
 
 const displayError = (error) => {
-  const message = `prettier-atom: ${error.toString()}`;
-  const detail = error.stack.toString();
-
-  atom.notifications.addError(message, { detail, dismissable: true });
+  atom.notifications.addError('prettier-atom failed!', {
```

Comment:
Hmm, yeah I'd either go with that or just plain `prettier-atom`? I'm okay with either, this change is so much better than what we've got now anyway lol